pkg/ui/distccl/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+# TODO (koorosh): genrule produces directory as output, should be changed later.
```

Comment:
Is there a bug filed for this?

Reply:
Thanks for reminder! 
https://github.com/cockroachdb/cockroach/issues/70348